tools/test-proxy/Azure.Sdk.Tools.TestProxy/Store/IAssetsStore.cs

```diff
@@ -0,0 +1,28 @@
+using System.IO;
+
+namespace Azure.Sdk.Tools.TestProxy.Store
+{
+    public interface IAssetsStore
+    {
+        /// <summary>
+        /// Given a configuration, push the changes made by the test-proxy into the remote store.
+        /// </summary>
+        /// <param name="pathToAssetsJson"></param>
+        /// <param name="contextPath"></param>
+        public abstract void Save(string pathToAssetsJson, string contextPath);
```

Comment:
Is `remote store` in this case just whatever local git/disk provider is being used, or does it mean "push to a shared upstream" remote?

Reply:
The latter; pushed to a shared upstream remote. This equivalent to an `Push` operation in the asset-sync script.

Reply:
In this case I'd be a little worried that `Save` does not communicate that effectively? Or is this just an internal name/route?

Reply:
What do you think will be more appropriate here?

I am thinking of `save` as "place in external storage" whether it's git or not. Make it `push`?